ajax/libs/algebra.js/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "algebra.js",
+  "version": "0.2.6",
+  "description": "Build, display, and solve algebraic equations.",
+  "filename": "algebra.min.js",
+  "scripts": {
+    "test": "make lint test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nicolewhite/algebra.js.git"
+  },
+  "files": [
+    "algebra.js",
+    "src/"
+  ],
+  "autoupdate" : {
```

Comment:
I think you can use two `npmFileMap` but different `basePath` to mention the file. One is `""` to grab the `algebra.js`, another is `"src"` to grab the files under `src/`, thanks!